drivers/SmartThings/zwave-switch/fingerprints.yml

```diff
@@ -399,7 +399,7 @@ zwaveManufacturer:
     deviceLabel: Evolve Switch
     manufacturerId: 0x0113
     productType: 0x5257
-    deviceProfileName: switch-binary
+    deviceProfileName: dimmer-switch
```

Comment:
this device does not support Switch Level

also there are three other fingerprints that should have a new profile in addition to this one: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-switch.src/zwave-switch.groovy#L24